server/routes/deleteObject.js

```diff
@@ -33,27 +34,29 @@ router.post('/:url', async function (req, res) {
     let objects = [];
     let stream = minioClient.listObjects(
       bucketName,
-      'hv/' + user + '/' + name + fileId,
+      'hv/' + user + '/' + name,
       true,
     );
     stream.on('data', async obj => {
       objects.push(obj.name);
     });
     stream.on('error', err => {
-      console.error('Error listing objects:', err);
+      log.error('Error listing objects:', err);
       res.status(500).json({ error: 'Failed to list objects' });
     });
     stream.on('end', async () => {
       minioClient.removeObjects(bucketName, objects, function (e) {
         if (e) {
-          return console.error('Unable to remove objects', e);
+          return log.error('Unable to remove objects', e);
         }
-        // console.error("removed the objects successfully")
+        log.info(
```

Comment:
The call to `delete_file` has changed the order of parameters. Ensure that the function definition for `delete_file` matches the new order of arguments being passed.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*